dev_env_setup.sh

```diff
@@ -12,145 +12,75 @@ cd "$SCRIPT_DIR"
 echo "Running 'yarn install' in the root directory..."
 yarn install
 
-# Step 2: Run yarn build in the root directory
-echo "Running 'yarn build' in the root directory..."
-yarn build
```

Comment:
originally `yarn build` fires the `rollup` to build all the UMDs/ESMs in packages... You switched to trying running `yarn build` inside each pkg. But does this produce the same output?
Taking package `packages/cubejs-client-core` as an example:
There is no `build` script in `package.json` and there is no `ts` files. So new version won't do anything, while original `rollup` will prepare UMD for it.

Reply:
fixed